cli/cli.ts

```diff
@@ -17,5 +18,6 @@ program
   .version("0.0.1")
 
 // Add commands here
+datasetDownloadCommand
```

Comment:
can you use the same pattern used in the tscircuit cli repo? There should be a `registerDatasetDownload(program)` function

https://github.com/tscircuit/cli/tree/main/cli